web_external/User/invitationConfirmationView.js

```diff
@@ -0,0 +1,27 @@
+isic.views.InvitationConfirmationView = isic.View.extend({
```

Comment:
Our convention is that view files get named with a leading uppercase letter. We also have a semi-convention of always having a (possibly blank) `.styl` file for every view / template group.

I'd be fine with reconsidering both conventions, but they should be followed for now.